federatedpersistence/src/main/java/org/openmbee/mms/federatedpersistence/permissions/FederatedPermissionUpdatesResponseBuilder.java

```diff
@@ -0,0 +1,76 @@
+package org.openmbee.mms.federatedpersistence.permissions;
+
+import org.openmbee.mms.core.builders.PermissionUpdatesResponseBuilder;
+import org.openmbee.mms.core.objects.PermissionUpdateResponse;
+import org.openmbee.mms.core.objects.PermissionUpdatesResponse;
+
+public class FederatedPermissionUpdatesResponseBuilder extends PermissionUpdatesResponseBuilder {
+
+    private Boolean inherit;
+    private Boolean isPublic;
+    private FederatedPermissionsUpdateResponseBuilder usersBuilder = new FederatedPermissionsUpdateResponseBuilder();
+    private FederatedPermissionsUpdateResponseBuilder groupsBuilder = new FederatedPermissionsUpdateResponseBuilder();
+
+    @Override
+    public FederatedPermissionUpdatesResponseBuilder setInherit(Boolean inherit) {
```

Comment:
setInherit, setPublic, insert functions are never called on this class, only on parent class, can be removed

Reply:
fixed in #260